### ISSUE TITLE
VICE is not launched with correct directory argument

### ISSUE THREAD

**Issue by KeithHenrickson** [empathicqubit/vscode-cc65-vice-debug#24]
In the VICE launch, the directory is set to:
`'-directory' '"c:\Users\KeithHenrickson\.vscode\extensions\entan-gl.cc65-vice-3.7.3\dist\system\DRIVES;c:\Users\KeithHenrickson\.vscode\extensions\entan-gl.cc65-vice-3.7.3\dist\system\PRINTER"'`

There's no mention of the C64 path in there. VICE dies with:
`C64MEM: Error - Couldn't load kernal ROM `kernal'.
Error - Machine initialization failed.`

I poked some through the javascript, and it seems like I need to somehow set a "machineType", to "C64", but I can't see any setting in launch.json or elsewhere that would enable that.

My project is just the assembly sample project from here.

**Comment by empathicqubit** [comment]
The machine type is inferred from the libraries that are included. If you look at `asm-project.c64.dbg` do you see 

```
lib id=0,name="/usr/share/cc65/lib/c64.lib"                                          
```

**Comment by KeithHenrickson** [comment]
Yeah, the line is there:
lib	id=0,name="C:\ProgramData\chocolatey\lib\cc65-compiler\tools\lib/c64.lib"

And that path does seem right:
PS C:\ProgramData\chocolatey\lib\cc65-compiler\tools\lib> dir


    Directory: C:\ProgramData\chocolatey\lib\cc65-compiler\tools\lib


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----

-a----         2/18/2021   7:06 PM        1137069 c64.lib

Unless Windows is complaining about the / instead of the \
Powershell doesn't seem to care.

But the VICE command is still off:
PS C:\Users\KeithHenrickson\Documents\VSCode\asm-project>  ${env:ELECTRON_RUN_AS_NODE}='1'; & 'C:\ProgramData\chocolatey\lib\winvice-nightly\tools\GTK3VICE-3.5-win64\bin\x64sc' '-directory' '"c:\Users\KeithHenrickson\.vscode\extensions\entan-gl.cc65-vice-3.7.3\dist\system\DRIVES;c:\Users\KeithHenrickson\.vscode\extensions\entan-gl.cc65-vice-3.7.3\dist\system\PRINTER"' '-remotemonitor' '-remotemonitoraddress' '127.0.0.1:29347' '-binarymonitor' '-binarymonitoraddress' '127.0.0.1:29864' '-autostart-warp' '-autostartprgmode' '1' '-autostart-handle-tde' '-initbreak' '2061' '-logfile' 'C:\Users\KEITHH~1\AppData\Local\Temp\cc65-vice-13996QLYBEAWYWlgY' '-moncommands' 'C:\Users\KeithHenrickson\Documents\VSCode\asm-project\asm-project.c64.lbl' '-model' 'ntsc'

**Comment by empathicqubit** [comment]
I'm working on deploying a fix in #25 . I'm having some issues testing against VICE latest that I'm trying to sort out.

**Comment by empathicqubit** [comment]
I think I fixed the issue in 3.7.4. I don't have a Windows machine to test, so please let me know if it seems to be working for you. I tried running in Github Actions, but the Windows variation of the build runs the tests too slowly and fails.

**Comment by empathicqubit** [comment]
I'm going to close this issue since you haven't responded in a little while. Please let me know if you have more problems and I'll reopen it.
